vcr_transport.go

```diff
@@ -38,13 +38,41 @@ func (t *vcrTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	t.PCB.Logger.Printf("INFO - Cassette '%s' - Executing request to live server for %s %s\n", t.Cassette.Name, req.Method, req.URL.String())
 	resp, err = t.PCB.Transport.RoundTrip(req)
 
-	if !t.PCB.DisableRecording {
-		// the VCR is not in read-only mode so
-		t.PCB.Logger.Printf("INFO - Cassette '%s' - Recording new track for %s %s as %s %s\n", t.Cassette.Name, req.Method, req.URL.String(), copiedReq.Method, copiedReq.URL)
-		if err := recordNewTrackToCassette(t.Cassette, copiedReq, resp, err); err != nil {
-			t.PCB.Logger.Println(err)
-		}
+	if err := t.recordNewTrackToCassette(copiedReq, resp, err); err != nil {
```

Comment:
This appears to copy the request without potential `SaveRequestFilters` being applied.

Reply:
That is applied inside the function, here: https://github.com/seborama/govcr/pull/44/files#diff-9d890bae345abd19516bee241635cca7R59